Frameworks/UIKit/ForceInclusion.mm

```diff
@@ -47,6 +48,7 @@ void ForceInclusion() {
     [UISwitch class];
     [UISlider class];
     [UIProxyObject class];
+    [RTObject class];
```

Comment:
Well, they really are RTObjects, after we convert them to RTObjects. But you're right that there isn't a lot of reason to do this. One thing that I'd like to do in the near future is eliminate any continued use of RTObject (primarily in the xamlElement property on UIViews) and instead dynamically load the appropriately-typed projection classes as they're needed.